engine_test.go

```diff
@@ -919,6 +956,23 @@ var queries = []struct {
 			{"f64"},
 			{"b"},
 			{"f"},
+			{"id"},
+			{"int8"},
+			{"int16"},
+			{"int32"},
+			{"int64"},
+			{"uint8"},
+			{"uint16"},
+			{"uint32"},
+			{"uint64"},
+			{"float32"},
+			{"float64"},
+			{"timestamp"},
```

Comment:
Some of these are reserved words. Might want to name them something slightly different to make things easier.